com.github.halfmexican.Mingle.json

```diff
@@ -0,0 +1,59 @@
+{
+    "app-id" : "com.github.halfmexican.Mingle",
```

Comment:
It should be io.github, the rules are right there in the PR checklist.

Reply:
I feel so dumb, sorry

Reply:
changed the entire App ID